packages/wdio-jasmine-framework/src/index.ts

```diff
@@ -351,7 +351,7 @@ class JasmineAdapter {
         return function (this: jasmine.Spec, passed: boolean, data: ResultHandlerPayload) {
             try {
                 expectationResultHandler!.call(this, passed, data)
-            } catch (e) {
+            } catch (e: any) {
```

Comment:
```suggestion
            } catch (e: Error) {
```

Reply:
Catch clause variable type annotation must be 'any' or 'unknown' if specified.